### PR TITLE
refactor: OBLITERATE teaserId prop, which is required though never used.

### PR DESCRIPTION
**Pull request by fabiosantoscode**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@economist/component-teaser",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
Comment (keithamus) [inline package.json]: No manual bumping please. Please provision. This doesn't seem so urgent as to not warrant a provision.

**Comment by umbobabo** [comment]
Please @keithamus @gcedo provision this component, the fact that I'm merging this will not affect you work.

**Comment by gcedo** [comment]
@umbobabo, @keithamus  I will provision it.

**Comment by keithamus** [comment]
:+1: thanks @gcedo

**Pull request merged**